<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>650</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #007dec;
	color: #fff;
	border-radius: 5px;
}

QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: #214a80;
	background-position: calc(100% - 10px)center;
}

QPushButton:hover {
	background-color: #214a80;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QLabel" name="login_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>410</width>
       <height>650</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1e2833;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="login_icon">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>90</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/login.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>365</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1e2833;
border: none;
border-bottom: 1px solid #434a52;
color: #fff;
padding-left: 3px;</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>495</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>425</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1e2833;
border: none;
border-bottom: 1px solid #434a52;
color: #fff;
padding-left: 3px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="no_acc_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>555</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1e2833;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_close_btn">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>15</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1e2833;
color: #fff;</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_username_alert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>405</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #aa0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="login_password_alert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>465</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #aa0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signup_page">
    <widget class="QLabel" name="signup_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>410</width>
       <height>650</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1e2833;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signup_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1e2833;
border: none;
border-bottom: 1px solid #434a52;
color: #fff;
padding-left: 3px;</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signup_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1e2833;
border: none;
border-bottom: 1px solid #434a52;
color: #fff;
padding-left: 3px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signup_confirm">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1e2833;
border: none;
border-bottom: 1px solid #434a52;
color: #fff;
padding-left: 3px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="signup_icon">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>65</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/register.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_close_btn">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>15</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1e2833;
color: #fff;</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="signup_username_alert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #aa0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="signup_password_alert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #aa0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="signup_confirm_alert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #aa0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>login_username</tabstop>
  <tabstop>login_password</tabstop>
  <tabstop>login_btn</tabstop>
  <tabstop>no_acc_btn</tabstop>
  <tabstop>login_close_btn</tabstop>
  <tabstop>signup_username</tabstop>
  <tabstop>signup_password</tabstop>
  <tabstop>signup_confirm</tabstop>
  <tabstop>signup_btn</tabstop>
  <tabstop>signup_close_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
